### ISSUE TITLE
Latest version of Lab is failing on es6 syntax?

### ISSUE THREAD

**Issue by visualjeff** [hapijs/lab#676]
Running with Node v6.9.1 and Hapi ^16.0.1
```
npm test

> ./node_modules/lab/bin/lab

/home/visualjeff/Documents/nodeHapiTemplate/index.js:25
const server = new Hapi.Server();
^^^^^
SyntaxError: Unexpected token const
```

Any tips on how to get around this??

Thanks in advance,

Jeff

**Comment by visualjeff** [comment]
At first, I thought this might explain why:  https://github.com/hapijs/lab/issues/668

But now after looking my _node_modules/lab/lib/coverage.js_ I notice:

```
    const tree = Espree.parse(content, {
        loc: true,
        comment: true,
        range: true,
        ecmaVersion: 8
    });
```
So I guess I'm out of luck...

**Comment by visualjeff** [comment]
lab version 11.2.2  works fine.
With coverage.js ecmaVersion set to 6.

**Comment by visualjeff** [comment]
Resolved

**Comment by Marsup** [comment]
Would be more helpful to let others know how you solved it.

**Comment by geek** [comment]
@visualjeff what is the solution?

**Comment by jeffm13** [comment]
Sounds like it was solved by downgrading to 11.2.1.  We're having trouble too.  Test cases run fine on Node v6.9.5 using lab 11.2.   The same tests will not run under Lab 12.1.  Each test fails with the same exception:  ecmaVersion must be 3, 5, 6, or 7.

**Comment by geek** [comment]
@jeffm13 can you provide an example test case that is failing

**Comment by kbmmdwork** [comment]
Like others (above) I downgraded to 11.x.x to get this working... then I found out that I could resolve it by pulling the latest version (3.4.3) of espree into my own app (npm install espree@latest --save-dev).  That's not great, but it manages to satisfy lab's requirement for (3.x.x).  This is a bit ridiculous though - IMO, lab itself should rather depend on the minimum version of espree that it really needs instead of making the "host" app do that.
